src/AutoRest.CSharp/Common/Input/CodeModelConverter.cs

```diff
@@ -379,6 +379,7 @@ private static InputType CreateType(Schema schema, string? format, IReadOnlyDict
             Namespace: schema.Extensions?.Namespace,
             Accessibility: schema.Extensions?.Accessibility,
             Description: schema.CreateDescription(),
+            Usage: InputModelTypeUsage.RoundTrip,
```

Comment:
For the swagger path, we won't generate models, so will this value be InputModelTypeUsage.None?

Reply:
No. We will support HLC from swagger. We will convert to the right Usage which is calculated by autorest and recorded in schema. And there is a flag in ModelGenerator which filter to generate model from swagger.